README.md

```diff
@@ -140,3 +142,40 @@ This allows to delete a specific reminder.
 
 * 204 if request succeeded.
 * 404 if no reminder with this ID exists.
+
+## API notifications
+
+### POST `/notifications`
```

Comment:
Renamed to `/subscriptions` so the API is clearer. We don't provide a way for client to send notifications.